### ISSUE TITLE
News not upating since 5 days now

### ISSUE THREAD

**Issue by amitnarw** [SauravKanchan/NewsAPI#29]

**Comment by tranhuudang** [comment]
It's finally backed ;)))) I'm happy to see it working again.

**Comment by amitnarw** [comment]
> It's finally backed ;)))) I'm happy to see it working again.

Old data was already there, but there is no new data after 21st April 2022

**Comment by tranhuudang** [comment]
You're right, the bot seem to commit data to the repos regularly but no new data come in.

**Comment by yashash1511** [comment]
still not updating :(

**Comment by tranhuudang** [comment]
We should build one, I'll start my own project next month.

**Comment by yashash1511** [comment]
> We should build one, I'll start my own project next month.

https://github.com/win11react/NewsAPI we made one feel free to use urself (you can configure thing according to your need from the workflow file)

**Comment by amitnarw** [comment]
> > We should build one, I'll start my own project next month.
> 
> https://github.com/win11react/NewsAPI we made one feel free to use urself (you can configure thing according to your need from the workflow file)

Nice, but it's for unlimited use? I mean Saurav's NewsAPI have no limits. But in his API news refresh every hour.

**Comment by yashash-pugalia** [comment]
> Nice, but it's for unlimited use? I mean Saurav's NewsAPI have no limits

yes unlimited just GitHub Pages sites have a soft bandwidth limit of 100 GB per month. (the same applies to Saurav's NewsAPI)

> But in his API news refresh every hour.

we are updating the news every day (Edit: now every hour) which is fine for our needs you can just change the cron job to make it every hour
maximum 10times each hour should work as GitHub Pages sites have a soft limit of 10 builds per hour.

https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#limits-on-use-of-github-pages

**Comment by yashash1511** [comment]
@amitnarw we updated it to run every hour

**Comment by tranhuudang** [comment]
Everyday update is fine for me, what I really need is a stable repository, that not easy shut down or be abandoned by the owner. I'll star your repos and working with your repos.

**Comment by yashash1511** [comment]
> Everyday update is fine for me, what I really need is a stable repository, that not easy shut down or be abandoned by the owner. I'll star your repos and working with your repos.

it will be stable as we are using it on some quite large projects like [blueedgetechno/win11React](https://github.com/blueedgetechno/win11React)

**Comment by tranhuudang** [comment]
That's what I mean, thank you so much for your work. I currently have nothing, but that's  must me a couple of cup of coffee for you when I'm in the right condition. So, a big thanks again.

**Comment by rushilg13** [comment]
[https://github.com/win11react/NewsAPI ](https://github.com/win11react/NewsAPI ) has stopped working it seems. Any other News API that's working currently?

**Comment by Ostabo** [comment]
If someone is still searchin for something similar to this, (only top-headlines tho)
I made a GitHub Actions based version at [https://github.com/Ostabo/news-api](https://github.com/Ostabo/news-api) 
You could fork it and change the countries to your needs or use the 8 I used
